frontend/src/containers/PlayerPage/PlayerPage.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import ReactPlayer from 'react-player';
+import { withRouter } from 'react-router';
+import TopBar from '../../components/TopBar/TopBar';
+import { DeleteIcon, DownloadIcon, InfoIcon } from '../../assets';
+import './styles.css';
+import { getVideoDetails } from '../../api/VideoAPI';
+
+class PlayerPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      url: undefined,
+      video: undefined,
+      screenWidth: window.innerWidth,
+      screenHeight: window.innerHeight,
+    };
+    this.topBarRef = React.createRef();
+  }
+
+  componentDidMount() {
+    window.addEventListener('resize', this.updateWindowDimensions);
+    const { match } = this.props;
+    const { videoId } = match.params;
+    getVideoDetails(videoId).then((response) => {
+      const userId = window.sessionStorage.getItem('id');
+      const url = `http://localhost:61346/api/Videos/stream?videoId=${videoId}&userId=${userId}`;
+      this.setState({ url, video: response.data });
+    });
```

Comment:
Catch an error here, because if we cannot load video information, then video player page just shows an empty screen, so we should notify a user with a top center snackbar that we could not load the video. On hide of snackbar we should go back to library page